packages/contracts/README.md

```diff
@@ -22,7 +22,6 @@ yarn build
 
 ## Running Tests
 
-
```

Comment:
You should add here that you need to install the submodules to run the tests or to compile

`git submodule init` and `git submodule update`